docs/tutorial/dark-mode.md

```diff
@@ -13,7 +13,7 @@ from the OS.
 
 If your app has its own dark mode, you should toggle it on and off in sync with
 the system's dark mode setting. You can do this by using the
-[prefer-color-scheme] CSS media query.
+[prefers-color-scheme] CSS media query.
```

Comment:
```suggestion
[prefers-color-scheme][] CSS media query.
```

?

Reply:
Not sure here. For full context for anyone following along, the docs are a mixed bag of reference links, since you can have three variants (in GFM):

* `[foo][]` - [collapsed reference link](https://github.github.com/gfm/#collapsed-reference-link) (link text with an empty link label, also known as "implicit link shortcut" in original Markdown spec)
* `[foo]` - [shortcut reference link](https://github.github.com/gfm/#shortcut-reference-link) (omits the empty link label)
* `[foo][bar]` - [full reference link](https://github.github.com/gfm/#full-reference-link) (link text followed by link label)

All three variants are used throughout the docs currently, the shortcut version is pretty common (like here). I don't have a strong opinion on style here, other than that the trailing `[]` does make it a bit more explicit, and that Markdown is a bit of a mess. 😅

Reply:
huh TIL

Reply:
To clarify, since the shortcut version is more common (I believe, I might be wrong) in our docs, I've preferred that style when updating links.